libraries/AP_MSP/AP_MSP_Telem_DJI.cpp

```diff
@@ -15,10 +15,13 @@
 
 #include <AP_Common/AP_FWVersion.h>
 #include <AP_Vehicle/AP_Vehicle_Type.h>
+#include <AP_Stats/AP_Stats.h>
 
 #include "AP_MSP.h"
 #include "AP_MSP_Telem_DJI.h"
 
+#include <stdio.h>
```

Comment:
probably not required

Reply:
should I use hal.util->snprintf() instead?

Reply:
Probably.  But we're 50/50-ish in the code so *shrug* ...